<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConferoClass</class>
 <widget class="QMainWindow" name="ConferoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Confero</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>0</y>
      <width>761</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QsciScintilla" name="textEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>508</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>0</y>
      <width>259</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="uniformRowHeights">
     <bool>true</bool>
    </property>
    <property name="expandsOnDoubleClick">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string notr="true">Documents</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>282</y>
      <width>259</width>
      <height>250</height>
     </rect>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>250</width>
        <height>171</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>174</y>
        <width>250</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionBash"/>
    <addaction name="actionBatch"/>
    <addaction name="actionCMake"/>
    <addaction name="actionC"/>
    <addaction name="actionC_2"/>
    <addaction name="actionCSS"/>
    <addaction name="actionD"/>
    <addaction name="actionDiff"/>
    <addaction name="actionFortran"/>
    <addaction name="actionFortran77"/>
    <addaction name="actionHTML"/>
    <addaction name="actionIDL"/>
    <addaction name="actionJava"/>
    <addaction name="actionJavaScript"/>
    <addaction name="actionLua"/>
    <addaction name="actionMakefile"/>
    <addaction name="actionPascal"/>
    <addaction name="actionPerl"/>
    <addaction name="actionPostScript"/>
    <addaction name="actionPOV"/>
    <addaction name="actionProperties"/>
    <addaction name="actionPython"/>
    <addaction name="actionRuby"/>
    <addaction name="actionSpice"/>
    <addaction name="actionSQL"/>
    <addaction name="actionTCL"/>
    <addaction name="actionTeX"/>
    <addaction name="actionVerilog"/>
    <addaction name="actionVHDL"/>
    <addaction name="actionXML"/>
    <addaction name="actionYAML"/>
    <addaction name="actionNone"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionBash">
   <property name="text">
    <string>Bash</string>
   </property>
  </action>
  <action name="actionBatch">
   <property name="text">
    <string>Batch</string>
   </property>
  </action>
  <action name="actionCMake">
   <property name="text">
    <string>CMake</string>
   </property>
  </action>
  <action name="actionC">
   <property name="text">
    <string>C++</string>
   </property>
  </action>
  <action name="actionC_2">
   <property name="text">
    <string>C#</string>
   </property>
  </action>
  <action name="actionCSS">
   <property name="text">
    <string>CSS</string>
   </property>
  </action>
  <action name="actionD">
   <property name="text">
    <string>D</string>
   </property>
  </action>
  <action name="actionDiff">
   <property name="text">
    <string>Diff</string>
   </property>
  </action>
  <action name="actionFortran">
   <property name="text">
    <string>Fortran</string>
   </property>
  </action>
  <action name="actionFortran77">
   <property name="text">
    <string>Fortran77</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="actionIDL">
   <property name="text">
    <string>IDL</string>
   </property>
  </action>
  <action name="actionJava">
   <property name="text">
    <string>Java</string>
   </property>
  </action>
  <action name="actionJavaScript">
   <property name="text">
    <string>JavaScript</string>
   </property>
  </action>
  <action name="actionLua">
   <property name="text">
    <string>Lua</string>
   </property>
  </action>
  <action name="actionMakefile">
   <property name="text">
    <string>Makefile</string>
   </property>
  </action>
  <action name="actionPascal">
   <property name="text">
    <string>Pascal</string>
   </property>
  </action>
  <action name="actionPerl">
   <property name="text">
    <string>Perl</string>
   </property>
  </action>
  <action name="actionPostScript">
   <property name="text">
    <string>PostScript</string>
   </property>
  </action>
  <action name="actionPOV">
   <property name="text">
    <string>POV</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionPython">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionRuby">
   <property name="text">
    <string>Ruby</string>
   </property>
  </action>
  <action name="actionSpice">
   <property name="text">
    <string>Spice</string>
   </property>
  </action>
  <action name="actionSQL">
   <property name="text">
    <string>SQL</string>
   </property>
  </action>
  <action name="actionTCL">
   <property name="text">
    <string>TCL</string>
   </property>
  </action>
  <action name="actionTeX">
   <property name="text">
    <string>TeX</string>
   </property>
  </action>
  <action name="actionVerilog">
   <property name="text">
    <string>Verilog</string>
   </property>
  </action>
  <action name="actionVHDL">
   <property name="text">
    <string>VHDL</string>
   </property>
  </action>
  <action name="actionXML">
   <property name="text">
    <string>XML</string>
   </property>
  </action>
  <action name="actionYAML">
   <property name="text">
    <string>YAML</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
 </resources>
 <connections/>
</ui>
